db_manager/augur_manager.py

```diff
@@ -275,6 +275,7 @@ def multiselect_startup(self):
         df_repo_git_id = df_search_bar.copy()
         df_repo_git_id = df_repo_git_id[["repo_git", "repo_id"]]
         self.repo_git_to_repo_id = pd.Series(df_repo_git_id.repo_id.values, index=df_repo_git_id["repo_git"]).to_dict()
+        self.repo_id_to_repo_git = {value: key for (key, value) in self.repo_git_to_repo_id.items()}
```

Comment:
Vectorize with:
self.repo_id_to_repo_git = pd.Series(df_repo_git_id.repo_git.values, index=df_repo_git_id["repo_id"]).to_dict()

I checked it on my branch and it worked.